python/python/glide/async_commands/core.py

```diff
@@ -2862,6 +2862,60 @@ async def xgroup_destroy(self, key: str, group_name: str) -> bool:
             await self._execute_command(RequestType.XGroupDestroy, [key, group_name]),
         )
 
+    async def xgroup_create_consumer(
+        self, key: str, group_name: str, consumer: str
```

Comment:
```suggestion
        self, key: str, group_name: str, consumer_name: str
```
We might as well be consistent here